service-tests/github.js

```diff
@@ -79,29 +80,29 @@ t.create('GitHub open issues raw')
 t.create('GitHub open issues by label is > zero')
   .get('/issues/badges/shields/service-badge.json')
   .expectJSONTypes(Joi.object().keys({
-    name: Joi.equal('service-badge issues'),
+    name: 'service-badge issues',
     value: Joi.string().regex(/^[1-9][0-9]*[kMGTPEZY]? open$/)
   }));
 
 t.create('GitHub open issues by label is > zero')
   .get('/issues/Cockatrice/Cockatrice/Easy%20Change.json')
   .inspectJSON()
```

Comment:
Would you mind removing this debugging output? ^^

Reply:
Done. If I'm not mistaken, it was the only one used in the service tests.

Reply:
Yea, you're right. It's the only one I've noticed in the debugging output. I, or someone else, must have left it in there by mistake.